pkg/utils/utils.go

```diff
@@ -197,6 +199,21 @@ func GetS3PETMFromParamsMap(params map[string]interface{}, logger logrus.FieldLo
 		Region: aws.String(region),
 	}))
 
+	s3Url, ok := GetStringFromParamsMap(params, "s3Url", logger)
+	if ok {
+		sess.Config.Endpoint = aws.String(s3Url)
+	}
+
+	pathStyle, ok := GetStringFromParamsMap(params, "s3ForcePathStyle", logger)
+	if ok {
+		if GetBool(pathStyle, false) {
+			sess.Config.S3ForcePathStyle = aws.Bool(true)
+			logger.Info("Setting s3ForcePathStyle to true")
+		} else {
+			logger.Infof("Got %s for s3ForcePathStyle, setting s3ForcePathStyle to false", pathStyle)
+		}
+	}
+
 	prefix, ok := params["prefix"].(string)
```

Comment:
somehow, is this one missing?

Reply:
Yes missing one line to initial sess.Config.S3ForcePathStyle as false, have added.